docs/README.rst

```diff
@@ -541,7 +541,8 @@ Supported Route Configurations
 
    - For Edge and Unsecured Route types, the default backend port is 80.
 
-   - To expose a Service on any other ports. **specify the port in the Route config's "Port" field**.
+   - To expose a Service on any other port, **specify the port in the Route config's "Port: TargetPort" field**.
```

Comment:
Change to "specify the port number..." and remove the nested bullet point.

Reply:
Fixed